### ISSUE TITLE
Attachment - Modal error does not appear after uploading an attachment larger then 24MB

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#29452]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.3.83-1

**Reproducible in staging?:** Yes

**Reproducible in production?:** No

**If this was caught during regression testing, add the test name, ID and link from TestRail:**

**Email or phone of affected tester (no customers):**

**Logs:** https://stackoverflow.com/c/expensify/questions/4856

**Expensify/Expensify Issue URL:**

**Issue reported by:** Applause - Internal Team

**Slack conversation:**

## Action Performed:
1. Go to https://staging.new.expensify.com/
2. Log in with any Gmail account
3. Navigate to a chat
4. Click on the + button in the compose box and upload an attachment that is larger than the 24 MB limit

## Expected Result:
The error modal appears immediately upon upload

## Actual Result:
Nothing happened

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] Windows: Chrome 
- [ ] MacOS: Desktop

## Screenshots/Videos
<details>
<summary>Android: Native</summary>

</details>

<details>
<summary>Android: mWeb Chrome</summary>

</details>

<details>
<summary>iOS: Native</summary>

</details>

<details>
<summary>iOS: mWeb Safari</summary>

</details>

<details>
<summary>Windows: Chrome </summary>

 

https://github.com/Expensify/App/assets/78819774/995e853c-b14f-405a-85ad-5ebf700746c2

Video with Production:

https://github.com/Expensify/App/assets/78819774/dcb11343-15d1-46ea-ad88-8fbeb57c9393

</details>

<details>
<summary>MacOS: Desktop</summary>

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by luacmartins** [comment]
@ishpaul777 are you able to work on a PR now?

**Comment by luacmartins** [comment]
Cool, it's all yours!

**Comment by jasperhuangg** [comment]
@ishpaul777 any updates on that PR?

**Comment by mountiny** [comment]
CPed to staging

**Comment by joekaufmanexpensify** [comment]
@ishpaul777 / @sobitneupane , mind taking care of the BZ checklist this week?

**Comment by joekaufmanexpensify** [comment]
@sobitneupane mind taking care of the BZ checklist here when you're back next week?

**Comment by joekaufmanexpensify** [comment]
Not overdue

**Comment by joekaufmanexpensify** [comment]
Yep! The BZ checklist just needs to be complete before we can issue payment.

**Comment by joekaufmanexpensify** [comment]
[Bumped in Slack](https://expensify.slack.com/archives/C02NK2DQWUX/p1698178787160919)

**Comment by joekaufmanexpensify** [comment]
Still pending BZ checklist

**Comment by sobitneupane** [comment]
> **BugZero Checklist:** The PR fixing this issue has been merged! The following checklist ([instructions](https://stackoverflow.com/c/expensify/questions/14574)) will need to be completed before the issue can be closed:
> 
> * [ ] The PR that introduced the bug has been identified. Link to the PR:

https://github.com/Expensify/App/pull/26219
> * [ ] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:

https://github.com/Expensify/App/pull/26219#issuecomment-1760483920
> * [ ] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:
> * [ ] Determine if we should create a regression test for this bug.

Yes.
> * [ ] If we decide to create a regression test for the bug, please propose the [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) steps to ensure the same bug will not reach production again.

**Comment by joekaufmanexpensify** [comment]
Great, ty! I will finish this up today.

**Comment by joekaufmanexpensify** [comment]
All set to issue payment. This one qualifies for speed bonus, so we need to issue the following payments: 

- @ishpaul777 - $187.50 ($125 for PR, and $62.50 for speed bonus). Paid via upwork. 
- @sobitneupane - $187.50 ($125 for C+ and $62.50 for speed bonus). Paid via NewDot. 
- Reported internally, so no reporting payment.

**Comment by joekaufmanexpensify** [comment]
@ishpaul777 $187.50 sent and contract ended!

**Comment by joekaufmanexpensify** [comment]
@sobitneupane please request $187.50 via NewDot and let us know here once that's complete!

**Comment by sobitneupane** [comment]
Requested payment on newDot.

**Comment by joekaufmanexpensify** [comment]
TY! Closing as this is all set.

**Comment by JmillsExpensify** [comment]
$187.50 payment approved for @sobitneupane based on summary above.
